examples/BikeSensorConverter/BikeSensorConverter.ino

```diff
@@ -0,0 +1,352 @@
+/**********************************************
```

Comment:
Not sure if you meant to commit your project to the examples. If you want to commit this to the repo we should do it not on the shifting branch since it uses multiple profiles :) The project is looking good, you will have to tell me more about what it does.

Reply:
This happened by mistake. Can you remove it from the pull request ? We can talk about adding it to the examples, when it is thoroughly tested.